### ISSUE TITLE
FreeBSD transport errors

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#159]
Here we can see from a node injector, EPD the following errors:

```
2017-03-09 09:59:57 info: boot - daemon sender initializedUse of uninitialized value $hostid in concatenation (.) or string
 at /opt/kronometrix/bin/sender line 589.
Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 912.

2017-03-09 10:00:18 info: make_dsid - name=www.wordpress.comepdweba20c20f4fb8185b247c0cae0269e7f97 => e2309b08-de72-54e1-82
15-e7a1e8dfa7e8Use of uninitialized value $hostid in concatenation (.) or string at /opt/kronometrix/bin/sender line 589.
Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 912.

2017-03-09 10:00:19 info: make_dsid - name=www.wordpress.comepdweb74f6509884b50ed5f08db3ee8ccb2dc2 => 65c0c75b-56b0-522c-ac
0a-cf46c8d1e2b7Use of uninitialized value $hostid in concatenation (.) or string at /opt/kronometrix/bin/sender line 617.
Use of uninitialized value $hostid in concatenation (.) or string at /opt/kronometrix/bin/sender line 617.
Use of uninitialized value $hostid in concatenation (.) or string at /opt/kronometrix/bin/sender line 589.
Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 912.
```

**Comment by sparvu** [comment]
hostid corrected under sender.

**Comment by sparvu** [comment]
but still we have issues from other part.

```

2017-08-28 13:46:51 info: make_dsid - name=www.rackspace.comepdweba20c20f4fb8185b247c0cae0269e7f97 => 0ca4caf4-f7b1-5826-bc23-ad0c86dfdc1bUse of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:51 info: make_dsid - name=www.rackspace.comepdweb74f6509884b50ed5f08db3ee8ccb2dc2 => 325751d2-130e-5776-8720-1da6a56f7ccdUse of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:51 info: make_dsid - name=www.rackspace.comepdwebb665222c808c5277f03cc7c540f36773 => 50fe4815-0a94-5fb3-a294-a2708f2bb76c
Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:52 info: make_dsid - name=cloud.fujitsu.comepdweba20c20f4fb8185b247c0cae0269e7f97 => 929c3e03-4f31-5475-aa6c-d75b3c2b215d
Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:52 info: make_dsid - name=cloud.fujitsu.comepdweb74f6509884b50ed5f08db3ee8ccb2dc2 => b4c59e55-dfff-514f-85bd-cc4e76826afb
Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:52 info: make_dsid - name=cloud.fujitsu.comepdwebb665222c808c5277f03cc7c540f36773 => 06903415-8524-5bfb-886d-82a9f4fbbb6bUse of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:53 info: make_dsid - name=www.joyent.comepdweba20c20f4fb8185b247c0cae0269e7f97 => 0e6b8fcf-fcbd-5b84-a1cf-2d7cdf468deaUse of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:53 info: make_dsid - name=www.joyent.comepdweb74f6509884b50ed5f08db3ee8ccb2dc2 => 643e03c0-649d-5429-a9e4-45a2dca2047eUse of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:53 info: make_dsid - name=www.joyent.comepdwebb665222c808c5277f03cc7c540f36773 => 1a36f7c2-f51b-59cf-a1d7-148df3b18f14Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:54 info: make_dsid - name=opennebula.orgepdweba20c20f4fb8185b247c0cae0269e7f97 => 940b9cbc-eea3-5b50-b730-c3aaea42f094Use of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:54 info: make_dsid - name=opennebula.orgepdweb74f6509884b50ed5f08db3ee8ccb2dc2 => ea567117-8765-578d-add7-68560d40ea0aUse of uninitialized value $idl in concatenation (.) or string at /opt/kronometrix/bin/sender line 941.

2017-08-28 13:46:54 info: make_dsid - name=opennebula.orgepdwebb665222c808c5277f03cc7c540f36773 => 78c97591-c3bd-58f6-ab81-96ff28713e2e
```

**Comment by sparvu** [comment]
fixed for 1.2.16
